src/lpf2hub.ts

```diff
@@ -548,12 +548,27 @@ export class LPF2Hub extends Hub {
                     this.emit("tilt", "TILT", this._lastTiltX, this._lastTiltY, this._lastTiltZ);
                     break;
                 }
+                case Consts.DeviceType.CONTROL_PLUS_GYRO: {
+                    const gyroX = Math.round(data.readInt16LE(4) * 7 / 400);
```

Comment:
The reported scaling value ` 2000 / 28571.42` (which is actually equal to 7 / 100) seemed to produce values too large, so I calibrated it using a 33/45 rpm (200/270dps) record player which showed the values where exactly 4 times off (hence `7 / 400`).
Also the reading seems to show non-calibrated values, maybe the library could provide calibration for that.